### DISCUSSION TITLE
UI overhaul

### DISCUSSION THREAD

**Discussion by ayush0477** [community/community#138216] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

i found it difficult to navigate through github options , a beta ui test would be great.

**Comment by ebndev** [comment]
Hi @ayush0477, thanks for being a part of the GitHub Community!

Would you mind providing more details on where you were navigating and what issues you encountered? That way we can ensure your feedback may be reviewed by the proper GitHub team.

**Reply by ayush0477** [reply]
it is like very mental memory depended hope you understand,
 you need to know and remember where things are in order to your work otherwise your lost.
